lib/src/bottom_sheets/cupertino_bottom_sheet.dart

```diff
@@ -288,13 +308,15 @@ class CupertinoScaffold extends StatefulWidget {
       containerBuilder: (context, _, child) => _CupertinoBottomSheetContainer(
         child: child,
         backgroundColor: backgroundColor,
+        topRadius: topRadius,
```

Comment:
Here instead of defining a new radius use the one set at the CupertinoScaffold

Reply:
done